modules/coredns/collect.go

```diff
@@ -3,30 +3,44 @@ package coredns
 import (
 	"fmt"
 
+	"github.com/hashicorp/go-version"
```

Comment:
Can we use

https://github.com/netdata/go.d.plugin/blob/9e912ff0bde663f2319f00bc315140d59c59fc91/go.mod#L11

?

Reply:
sure yes